.github/workflows/workflow-vercel-deployment.yml

```diff
@@ -16,18 +27,14 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Install Vercel CLI
         run: npm install --global vercel@latest
       - name: Pull Vercel Environment Information
         run: vercel pull --yes --environment=preview --token=${{ secrets.VERCEL_TOKEN }}
       - name: Build Project Artifacts
         run: vercel build --token=${{ secrets.VERCEL_TOKEN }}
       - name: Deploy Project Artifacts to Vercel
-        run: vercel deploy --prebuilt --token=${{ secrets.VERCEL_TOKEN }}
-        
-      - name: Comment on PR
```

Comment:
why was the comment on PR removed? Was it already displayed?

Reply:
I found it useless because it was a set url that feature was implemented to replace vercel's bot which comment the preview URL (not static or whatever the term is not sure) but I'll put it back if you liked it

Reply:
can you show me a screenshot or link to an example before/after this change? how do we find out which URL it is deployed to now?

Reply:
it's from the logs there's not really a before after I just figured since the everyone knows the URL and the bot tend to get annoying commenting each time you push to the branch it creates a bunch of comments and it can make it hard to follow discussions I will try to implement it back but only on PR creation

Reply:
I added it back